javascript-forms/form-events.html

```diff
@@ -37,5 +37,6 @@ <h2>Contact Us</h2>
     </form>
   </div>
   <script src="form-events.js"></script>
+  <script src="form-values.js"></script>
```

Comment:
You shouldn't need to add this. You already have a `form-values.html` where this script tag is loaded

Reply:
![Screenshot 2023-04-05 at 12 06 20 PM](https://user-images.githubusercontent.com/121325991/230180244-3435e232-e58d-4f0c-b6f5-f583ebfb956e.png)